UI/EncyclopediaDetailPanel.cpp

```diff
@@ -71,9 +71,23 @@ namespace {
     const std::string TAG_ALWAYS_REPORT = "CTRL_ALWAYS_REPORT";
     /** @content_tag{CTRL_EXTINCT} Added to both a species and their colony building.  Handles display in planet suitability report. **/
     const std::string TAG_EXTINCT = "CTRL_EXTINCT";
+    /** @content_tag{PEDIA_} Prefix to define a category for the article of the containing content definition. **/
+    const std::string TAG_PEDIA_PREFIX = "PEDIA_";
 }
 
 namespace {
+    /** Checks content tags for pedia category, returns category of first match */
```

Comment:
elaborate much more on this. I don't understand what it's doing from this comment.